h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostUtils.java

```diff
@@ -113,17 +114,30 @@ public static DMatrix convertFrameToDMatrix(Key<DataInfo> dataInfoKey,
             int cols = di.fullN();
             float[][] data = allocateDenseMatrix(nRows, di);
             long actualRows = denseChunk(data, chunks, f, vecs, w, di, cols, resp, weights, f.vec(response).new Reader());
+
+            Timer timer = new Timer();
```

Comment:
@michalk: Does it make any difference in speed if we use long start = System.currentTimeMillis(); and then log the difference as long timeTaken = System.currentTimeMillis()-start?